server/views/auth/login.handlebars

```diff
@@ -71,14 +71,14 @@
         </div>
       </div>
       <div class="col-md-12 position-relative">
-        {{!--
-        <label for="validationEmailField" class="form-label">Email</label> --}}
+        
+        <label for="validationReEnterPasswordField" class="form-label" >Password validation</label>
```

Comment:
can we double check if form is submitted with by testing for wrong passwords etc.

Reply:
Wrong in put still throws an error here